src/users/users.service.ts

```
@@ -2,11 +2,7 @@ import { Injectable } from '@nestjs/common';
 
 @Injectable()
 export class UsersService {
-
-    async registerUser(){
-
-    }
-    async login(){
-        
-    }
+  async registerUser():Promise<any> {}
```

ESLint

[error] Insert `·`

(prettier/prettier)